gateway/src/apicast/policy/logging/readme.md

```diff
@@ -0,0 +1,175 @@
+# Logging policy
```

Comment:
@porueesq  Can you review this file please? I guess you'll use it as the base for your docs.

Reply:
@davidor Yes, I'll review the file.
Just for the record, have you updated the functionality of the Logging policy? We currently have this documented:
https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.5/html/administering_the_api_gateway/apicast_policies#logging_policy

Reply:
The functionality introduced in this PR will be available in 3scale 2.7 @porueesq